include/cblox/core/tsdf_submap_collection.h

```diff
@@ -79,10 +90,13 @@ class TsdfSubmapCollection {
   // Gets the tsdf submap associated with the passed ID
   bool getAssociatedTsdfSubMapID(const SubmapID submap_id,
                                  SubmapID *submap_id_ptr) const;
+  bool getTsdfSubmapConstPtrById(const SubmapID submap_id,
+                                 TsdfSubmap::ConstPtr &submap_const_ptr) const;
 
   // Interacting with the submap poses
   bool setSubMapPose(const SubmapID submap_id, const Transformation &pose);
   void setSubMapPoses(const TransformationVector &transforms);
+  bool getSubMapPose(const SubmapID submap_id, Transformation &pose) const;
```

Comment:
Output parameters should be pointers. I.e:

`Transformation* pose`

Reply:
Thanks. I'll use a pointer and indicate failure by setting it to nullptr (instead of the bool return value).